### ISSUE TITLE
Cannot See the labels from the xtest data for further evaluation

### ISSUE THREAD

**Issue by Symbadian** [yukitsuji/kitti-py#43]
hi @yukitsuji, this is not an issue I am hoping, but rather a question, please?
 
 I am trying to view the data from the test set to see what data type is challenge for the 3dcnn but unfortunately I didn't see that therein or maybe I missed it.

I have 80 videos of two classes
when I do the split xtrain, ytrain, xtest, ytest, 
X_train, Xval_test, Y_train, Yval_test = train_test_split(
        X, Y, train_size=0.8, test_size=0.2, random_state=1, stratify=Y, shuffle=True)
        
Is there a method to visually see the data (not numerically) the actual video or image labels?
for example xtest = vid12, vid24 ,vid11 , vid14, vid29 etc..
its the labels of the video therein xtest I am try to identify.

Why? this would tell me which videos are actually challenging the model and which is not!!
The confusion matrix is stating numerical outputs as well but I still have no way of identifying what videos are challenging the model even if it state 4 misclassifications, there's no way of know those 4 videos in the test. My analysis is blind

Can you guide me with this please?
